project/.gitattributes.twig

```diff
@@ -0,0 +1,2 @@
+.* export-ignore
+*.md export-ignore
```

Comment:
When removing all `Upgrade*.md` and `Readme.md` files, we should also remove the docs.